getnacelle/nacelle-js
# chore: update supported engines

## Why are these changes introduced?


- Updates the root of `nacelle-js` to support `node 18` which is now lts (also pins a volta version for devs using volta)


## How to Test

1. Use volta/nvm/<your-node-manager> to install node 18 (volta does this when you open the folder)
2. Run `npm run clean` to clean all of your node_modules
3. Run `npm run bootstrap` to bootstrap the packages
4. Run `npm run test:ci` to make sure all of the test pass
5. Run `npm run build` to make sure all of the projects build


## Checklist
